dsc/workflows/metadata_mapping/sccs.json

```diff
@@ -0,0 +1,79 @@
+{
+    "item_identifier": {
+        "source_field_name": "item_identifier",
+        "language": null,
+        "delimiter": "",
```

Comment:
Any reason why these are all empty strings `""`?  Could they be JSON `null`?  or omitted entirely?

Reply:
@ehanson8 @ghukill Please see the changes in the latest commit https://github.com/MITLibraries/dspace-submission-composer/pull/77/commits/72fef3098e137385a1d680b9ab2fd184ed7a8c97.